app.py

```diff
@@ -38,7 +38,48 @@ def mirror(name):
     }
     return create_response(data)
 
-# TODO: Implement the rest of the API here!
+@app.route('/users', methods = ['GET', 'POST'])
+def users():
+    if request.method == 'GET':
+        if request.args.get('team') is None:
+            data = {
+                'users': db.get('users')
+            }
+        else:
+            matched = [i for i in db.get('users') if i['team'] == request.args.get('team')]
+            data = {
+                'users': matched
+            }
+        return create_response(data)
+    elif request.method == 'POST':
+        input = request.get_json()
+        payload = {'name': input.get('name'),
+                   'age': input.get('age'),
+                   'team': input.get('team')}
+        return create_response(db.create('users', payload))
```

Comment:
Should check to see if `name`, `age`, and `team` are in received JSON. You'll get an exception if one or more of these aren't present, and you should throw back a 422 if that is the case